source/win-main/MainSidebar.vue

```diff
@@ -525,6 +546,11 @@ body {
         cursor: pointer;
         &:hover { text-decoration: underline; }
       }
+
+    }
+
+    div.toc-active {
+      background-color: rgb(200, 200, 200);
```

Comment:
Maybe instead of using the background color and put this in line with the rest of the application, apply a bold text weight and set the color to `var(--system-accent-color)`!

Reply:
Done.

Also, I moved the `toc-entry-active` (how it's called now) into to the `toc-entry-container` block. Before that, it was floating kinda on its own.